### ISSUE TITLE
Latest version error

### ISSUE THREAD

**Issue by javi-on** [facebook/regenerator#613]
Last night compile came with this error
```
[0m[91mnpm[0m[91m [0m[91mERR! Could not install from "node_modules/regenerator/packages/preset" as it does not contain a package.json file.
```

**Comment by brokenhoax** [comment]
^^Ran in to the same issue here. Any chance a fix is in the works?

**Comment by javi-on** [comment]
We have to set the previous version fixed in packge.json in order to bypass this thing

**Comment by brokenhoax** [comment]
yeah, I just thought of that ... pin an older version. thank you!

**Comment by CRCH** [comment]
It seems to be an issue after the last update 0.14.8
as a quick workaround add 
```
"resolutions": {
    "regenerator": "0.14.7"
  },
```
to your package.json

**Comment by CRCH** [comment]
It seems to be an issue after the last update 0.14.8
as a quick workaround add
```
"resolutions": {
    "regenerator": "0.14.7"
  },
```
to your package.json

**Comment by benjamn** [comment]
This should be fixed in `regenerator@0.14.9` (published to npm just now). Thanks everyone for reporting!
